generate_base_client.py

```diff
@@ -51,6 +51,12 @@
     type=str,
     help='Path to format output.',
 )
+_cmdline_parser.add_argument(
+    '-fs',
+    '--format-script-dir',
```

Comment:
consider using `-path` here to keep the arguments consistent across the other arguments. since it needs to be a dir, you can add [this](https://stackoverflow.com/a/51212150) kind of thing as a validator for the argument.